content_schemas/dist/formats/content_block_contact/notification/schema.json

```diff
@@ -415,7 +415,7 @@
         "change_history": {
           "$ref": "#/definitions/change_history"
         },
-        "contact_forms": {
+        "contact_links": {
```

Comment:
It might be helpful to include info about the change in the commit message -- e.g . something along 
> ...changes the name of the `contact_forms` object to `contact_links`